src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Microsoft.DotNet.Scaffolding.Core.csproj

```diff
@@ -10,4 +10,9 @@
     <PackageReference Include="System.CommandLine.NamingConventionBinder" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)src\dotnet-scaffolding\Microsoft.DotNet.Scaffolding.Internal\CliHelpers\*.cs">
```

Comment:
I wonder if we're trying to hard to avoid having Core reference .Internal. But for now it's fine, we can always revisit later.

Reply:
sounds good ty